src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.NormalSearch.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.PatternMatching;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.CodeAnalysis.Storage;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.NavigateTo
+{
+    internal abstract partial class AbstractNavigateToSearchService
+    {
+        public async Task SearchDocumentAsync(
```

Comment:
thse are all moves.  but i wanted all the OOP scaffolding to be not intermixed with the main logic.